index.bs

```diff
@@ -1432,10 +1432,10 @@ This is a [=static command=].
    <dt>Command Type</dt>
    <dd>
       <pre class="cddl remote-cddl">
-      session.Status = {
+      session.Status = (
```

Comment:
Can you explain why those were changed from `{` to `(`

Reply:
Because `{` are used for types, not groups.

Reply:
More specifically, `SessionCommand` is a set of group choices (not type choices because it uses `//`), so its constituents need to be groups, i.e. rules defined with `(`.

The reason `SessionCommand` **must** be a set of group choices is because it later gets spread into `Command` via `CommandData`. Only a set of group choices are allowed to be spread, not a type (like `{`).